Chiazokam/Politico
# #163355959 create the delete party endpoint

#### What does this PR do?
This PR tests and creates the delete party endpoint

#### Description of Task to be completed?
- write tests for all the functions
- create the endpoint

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-datastructures-delete-party-163355959`
- Start the server `npm run start:dev`
- Open `Postman`
- On `postman` Make a `DELETE` request to `http://localhost:3000/api/v1/parties/1 `
- You should receive a successful response

#### Any background context you want to provide?
Nil

#### What are the relevant pivotal tracker stories?
[#163355959](https://www.pivotaltracker.com/story/show/163355959)

####  Images

![image](https://user-images.githubusercontent.com/26940294/51785436-05d20780-2158-11e9-99fd-dadf1cb60ebc.png)
